pkg/minishift/cache/oc_caching_test.go

```diff
@@ -24,11 +24,246 @@ import (
 	"net/http"
 	"os"
 	"path/filepath"
+	"runtime"
 	"testing"
 )
 
-var testDir string
-var testOc Oc
+var (
+	openShiftReleaseTagInfo = `{
```

Comment:
It would have been good if we extract this into separate json file and read it. 
I am not a big fan of embedding such a long testdata. 
Also, this would help in making this testdata as reusable (if in case).